src/Microsoft.Health.Fhir.Core.UnitTests/Features/Search/SearchValues/DateTimeSearchValueTests.cs

```diff
@@ -83,7 +83,7 @@ public void GivenAPartialDateTime_WhenInitialized_ThenCorrectStartDateTimeAndEnd
         public void GivenAPartialStartDateTime_WhenInitialized_ThenCorrectStartDateTimeShouldBeAssigned(string input, string start)
         {
             PartialDateTime inputDateTime = PartialDateTime.Parse(input);
-            PartialDateTime endDateTime = new PartialDateTime(2018);
```

Comment:
I made this constructor `private` - previously it was only referenced by unit tests.